### ISSUE TITLE
Docklands - issue with waypoint on 6.9km/4.2mi

### ISSUE THREAD

**Issue by murech** [gtbikev/courses#27]
On docklands course, waypoint on 6.9km/4.2m causes issue for some users. GPS navigation redirects user to right instead of straight road. A possible cause for this behavior might be the change of obstacle detection released with version 0.3.0.2.

**Comment by murech** [comment]
Changed waypoints between 6.9 - 8.0 km. Bugfix implemented: 

|Course Name|Mod Release|Status
|:---|:---|:---
|road_docklands|v0.3.0.2|Not Tested

https://github.com/gtbikev/courses/blob/bugfix/road_docklands_0302/courses/road_docklands.json

Needs to be tested. Contributions welcomed. Please use the following testing setup:
|Mod Release|Auto Steering|Obstacle Detection|
|:---|:---|:---
|0.3.0.2|On|On|

Please report smart trainer model and ini file configuration in test report. For example:

**Test Setup**
Wahoo Kickr Core, 2018
`[MAIN]
SELECTEDBIKE = TRIBIKE
SLOPESCALE = 0.5
DEBUGWINDOW = False
INITIALGPSPOINTLAT = -19.106371
INITIALGPSPOINTLONG = -169.870977
FECDEVICEID = 0
IMPERIAL = False
USERWEIGHTKG = 75`

**Test Result**
...

**Comment by murech** [comment]
**Test Setup**
`Wahoo Kickr Core, 2018`
Mods: `GT Bike V (0.3.0.2)`, `Map Editor (2.13)`
INI:
```
SLOPESCALE = 0.25
FECDEVICEID = 0
PWRDEVICEID = 0
HRDEVICEID = 0
CADDEVICEID = 0
CONTROLSDEVICEID = 0
IMPERIAL = False
USERWEIGHTKG = 76
ROADFEEL = False
WAYPOINTDEFAULTCOLOR = FFADFF2F
WAYPOINTDEFAULTRADIUS = 20
ALWAYSUSEVIRTUALSPEED = False
```

**Test Result**
Issue resolved at waypoint 6.9km/4.2mi
